src/main/resources/xslt/inputs/ddi/source-fixed.xsl

```diff
@@ -1,771 +1,757 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file is used by all the DDI-input based transformations. We can't accept a full delete&remplace commits on this kind of files. We're loosing all the feedbacks of the previous commits and loose the ability to identify what evolutions comes from which commit. These files should be modified carefuly whith thin commits.